packages/backend/server/src/core/quota/schema.ts

```diff
@@ -93,11 +93,35 @@ export const Quotas: Quota[] = [
       memberLimit: 3,
     },
   },
+  {
+    feature: QuotaType.FreePlanV1,
+    type: FeatureKind.Quota,
+    version: 4,
+    configs: {
+      // quota name
+      name: 'Free',
+      // single blob limit 10MB
+      blobLimit: 10 * OneMB,
+      // server limit will larger then client to handle a edge case:
+      // when a user downgrades from pro to free, he can still continue
+      // to upload previously added files that exceed the free limit
+      // NOTE: this is a product decision, may change in future
+      businessBlobLimit: 100 * OneMB,
+      // total blob limit 10GB
+      storageQuota: 10 * OneGB,
+      // history period of validity 7 days
+      historyPeriod: 7 * OneDay,
+      // member limit 3
+      memberLimit: 3,
+      // copilot action limit 10
+      copilotActionLimit: 10,
```

Comment:
it should be 30 if I remember correctly?

Reply:
see https://whimsical.com/affine-ai-Jt3idZd6njecW3ZLFwHjFe